label_studio/data_import/functions.py

```diff
@@ -48,7 +48,8 @@ def async_import_background(
 
     if project_import.preannotated_from_fields:
         # turn flat task JSONs {"column1": value, "column2": value} into {"data": {"column1"..}, "predictions": [{..."column2"}]
-        tasks = reformat_predictions(tasks, project_import.preannotated_from_fields, project)
+        raise_errors = flag_set('fflag_feat_utc_210_prediction_validation_15082025', user=user)
```

Comment:
please don't hate me but I think this one needs to be
```suggestion
        raise_errors = flag_set('fflag_feat_utc_210_prediction_validation_15082025', user=project.organization.created_by)
```

Reply:
(here and below)

Reply:
no worries! I'll change it